packages/calcite-components/src/components/button/button.tsx

```diff
@@ -266,7 +265,6 @@ export class Button
     return (
       <InteractiveContainer disabled={this.disabled}>
         <Tag
-          aria-disabled={childElType === "a" ? toAriaBoolean(this.disabled || this.loading) : null}
```

Comment:
Not needed because the `InteractiveContainer` will handle setting `aria-disabled`

Reply:
Need to discuss if `this.loading` should be setting `aria-disabled` or some other aria attribute? @geospatialem

Reply:
Should this use `aria-busy` instead? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy

Reply:
The way I'm reading it, `aria-busy` would go on the content that's being updated (triggered by the loading button), but I could be wrong.

Reply:
@driskull What does the `Tag` refer to? When testing the component with `loading` the component is no longer keyboard accessible, and doesn't provide context that it's loading. It looks like the `disabled` attribute is applied, is that our intent?

Reply:
Tag revers to either a native button element or a anchor element. (`<button>` or `<a>`)

Currently, we have this as well:

```
disabled={childElType === "button" ? this.disabled || this.loading : null}
```

So maybe we shouldn't set it to disabled when loading?

Reply:
> So maybe we shouldn't set it to disabled when loading?

I thought we had set this previously, but maybe it somehow got changed in the recent past? We added a [recommendation to the support recently](https://developers.arcgis.com/calcite-design-system/components/button/#recommendations) on our documentation page.

Reply:
Yeah I would expect loading and disabled to be decoupled. It should be left up to the user to set both if needed.